website/docs/developer/open-attestation-cli.md

```diff
@@ -0,0 +1,677 @@
+---
+id: open-attestation-cli
+title: Open Attestation (CLI)
+sidebar_label: Open Attestation (CLI)
+---
+
+This CLI tool in the [Open Attestation CLI](https://github.com/Open-Attestation/open-attestation-cli) repository turns .json documents into any open-attestation verifiable documents. It applies the OpenAttestation algorithm to produce a hash of the json document and then creates a file with the data and proof of integrity.
+
+## Installation
+
+### Binary
+
+To install the binary, simply download the binary from the [CLI release page](https://github.com/Open-Attestation/open-attestation-cli/releases) for your OS.
+
+> We are aware that the size of the binaries must be reduced and we have tracked the issue in [Github](https://github.com/Open-Attestation/open-attestation-cli/issues/68). We hope to find a solution in a near future and any help is welcomed.
+
+### NPM
+
+Alternatively for Linux or MacOS users, if you have `npm` installed on your machine, you may install the cli using the following command:
+
+```bash
+npm install -g @govtechsg/open-attestation-cli
+```
+
+The above command will install the open-attestation CLI to your machine. You will need to have node.js installed to be able to run the command.
+
+You can also opt to use npx:
+
+```bash
+npx -p @govtechsg/open-attestation-cli open-attestation <arguments>
+```
+
+> In all the guides, we will refer to the CLI as `open-attestation` when running a command. That means we will assume the CLI is available in your execution path. If it's not the case, you will to change `open-attestation` by the full path to the executable.
+
+---
+
+## Usage
+
+#### Table of Contents
+
+- [Wrapping document](#wrapping-documents)
+- [Document privacy filter](#document-privacy-filter)
+- [Encrypting document](#encrypting-document)
+- [Decrypting document](#decrypting-document)
+- [Token registry](#token-registry)
+  - [Deploy](#deploy)
+  - [Issue](#issue)
+- [Document store](#document-store)
+  - [Deploy](#deploy-1)
+  - [Issue](#issue-1)
```

Comment:
nice these are in their own respective headers now, able to be explicit?
```suggestion
- [Token registry](#token-registry)
  - [Deploy](#deploy-token-registry)
  - [Issue](#issue-token-registry)
- [Document store](#document-store)
  - [Deploy](#deploy-document-store)
  - [Issue](#issue-document-store)
```

Reply:
ok done.